types.go

```diff
@@ -1,5 +1,10 @@
 package spec
 
+type SSZMarshaller interface {
+	MarshalSSZ() ([]byte, error)
+	UnmarshalSSZ(buf []byte) error
+}
```

Comment:
I think if we import this to `ssv-dkg` it is lenss nice that the interface is defined twice

Reply:
but we can probably live with it